osu.Game/Overlays/Dashboard/Home/HomePanel.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Extensions.Color4Extensions;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Effects;
+using osu.Framework.Graphics.Shapes;
+using osuTK;
+using osuTK.Graphics;
+
+namespace osu.Game.Overlays.Dashboard.Home
+{
+    public class HomePanel : Container
```

Comment:
perhaps should be `abstract` as I don't think anything should be instantiating this outright

Reply:
I'm using this in `HomeShowMoreNewsPanel` and will be used as a container for beatmaps later